tests/test_quickprod.py

```diff
@@ -0,0 +1,41 @@
+from pyscipopt import Model, quickprod
+from pyscipopt.scip import CONST
+from operator import mul
+
+def test_quickprod_model():
+    m = Model("quickprod")
+    x = m.addVar("x")
+    y = m.addVar("y")
+    z = m.addVar("z")
+    c = 2.3
+
+    q = quickprod([x,y,z,c]) == 0.0
+    s = reduce(mul,[x,y,z,c],1) == 0.0
```

Comment:
`reduce()` is not available in python3. The code must comply to both standards. Could you please rewrite this?

https://stackoverflow.com/a/13638960/673271

Reply:
This is also why the python3 test does not succeed.